### DISCUSSION TITLE
Sidebar appears on homepage after module upgrade

### DISCUSSION THREAD

**Discussion by bdrif** [imfing/hextra#480] [Q&A]
Hey everyone!

I'm hoping to get some help with an issue I've run into after upgrading the Hextra module to its latest version.

- I’ve got a sidebar navigation showing up on my homepage, which I’d love to get rid of. I tried adding `sidebar: exclude: true` in my `_index.md`, but that didn’t seem to do the trick.

![image](https://github.com/user-attachments/assets/e19b88bf-fafc-4ca2-b430-61ae904945e4) 
![image](https://github.com/user-attachments/assets/9c44fce9-eb7a-47a4-aed4-ea21530073e9)

Here's the content of my _index.md file for reference:
![image](https://github.com/user-attachments/assets/cf09952c-a571-4c67-930d-63b33f467e82)

- Another thing I’d like to tweak is how the left sidebar is diving too deep into the page structure:

![image](https://github.com/user-attachments/assets/b01b737a-1d41-4c05-a1e3-9ea379dd1c3b)

I already have a great “On this page” right sidebar, so I don’t need the left sidebar going into that level of detail.
If anyone has run into similar issues or knows how to handle this, I’d appreciate any pointers!

Thanks in advance for the help 😊

**Comment by imfing** [comment]
that sounds odd, did you follow the starter template to set up the project?
